plugins/inputs/t128_lte/t128_lte.go

```diff
@@ -0,0 +1,340 @@
+	return configData.Init.Id, nil
+}
+
+func (plugin *T128Lte) getLteInterfaces() <-chan map[string]interface{} {
+
+	lteInterfaces := make(chan map[string]interface{})
+	node, err := plugin.getNode()
+	if err != nil {
+		fmt.Sprintf("Unable to find node name %s : %w", node, err)
+		return lteInterfaces
+	}
+	deviceInterfaces, err := plugin.getDeviceInterface(node)
+	if err != nil {
+		return lteInterfaces
+	}
+
+	go func() {
+		defer close(lteInterfaces)
+
+		for _, deviceInterface := range deviceInterfaces {
+			fmt.Sprintf("Scanning device-interface %s", deviceInterface.(map[string]interface{})["type"].(string))
+			if deviceInterface.(map[string]interface{})["type"].(string) == "lte" {
+				lteInterfaces <- deviceInterfaces
+			}
+		}
+	}()
+
+	return lteInterfaces
+}
+
+func (plugin *T128Lte) getNode() (string, error) {
+	content, err := os.ReadFile(T128_CONFIG_FILE)
+	if err != nil {
+		return "", fmt.Errorf("Cannot read the %s : %w", content, err)
+	}
+	err = json.Unmarshal(content, &nodeData)
+	if err != nil {
+		return "", err
+	}
+	for _, node := range nodeData.Router {
+		for _, name := range node.Node {
+			if name.Name == plugin.NodeName {
+				return name.Name, nil
+			}
+		}
+	}
+	return "", fmt.Errorf("Unable to find node name")
+}
+
+func (plugin *T128Lte) getLteStateInfo(deviceName map[string]interface{}) (map[string]interface{}, error) {
+
+	targetName, ok := deviceName["target-interface"].(string)
+	if !ok {
+		targetName, _ = deviceName["targetInterface"].(string)
+	}
+	stateFile := filepath.Join(LTE_PATH_DIR, fmt.Sprint(targetName, ".state"))
+	_, err := plugin.checkFileExists(stateFile)
+	if err != nil {
+		return nil, err
+	}
+	lteStateInfo, _ := ioutil.ReadFile(stateFile)
+	var data map[string]interface{}
+	err = json.Unmarshal(lteStateInfo, &data)
+	return data, nil
+}
+
+func (plugin *T128Lte) getLteInfo(deviceName map[string]interface{}) (map[string]interface{}, error) {
+	targetName, ok := deviceName["target-interface"].(string)
+	if !ok {
+		targetName, _ = deviceName["targetInterface"].(string)
+	}
+	infoFile := filepath.Join(LTE_PATH_DIR, fmt.Sprint(targetName, ".info"))
+	_, err := plugin.checkFileExists(infoFile)
+	if err != nil {
+		return nil, err
+	}
+	lteInfo, _ := ioutil.ReadFile(infoFile)
+	var data map[string]interface{}
+	err = json.Unmarshal(lteInfo, &data)
+	return data, nil
+}
+
+func (plugin *T128Lte) checkFileExists(filePath string) (fs.FileInfo, error) {
+	checkFile, err := os.Stat(filePath)
+	if err != nil {
+		return nil, fmt.Errorf("LTE %s doesn't exists : %w", filePath, err)
+	}
+	return checkFile, nil
+}
+
+func (plugin *T128Lte) getDeviceInterface(name string) (map[string]interface{}, error) {
+	content, err := os.ReadFile(T128_CONFIG_FILE)
+	if err != nil {
+		return nil, fmt.Errorf("Cannot read the %s : %w", content, err)
+	}
+	err = json.Unmarshal(content, &deviceInterfaceDetails)
+	if err != nil {
+		return nil, err
+	}
+	for _, node := range deviceInterfaceDetails.Config.Authority.Router {
```

Comment:
Your for loop variable names are wrong. `node` -> `router`; `deviceInterface` -> `node`; `deviceType` -> `deviceInterface`.